### ISSUE TITLE
"needless argument: --debug=None" when running Serverspec tests

### ISSUE THREAD

**Issue by techraf** [metacloud/molecule#230]
Seemingly after upgrade to Molecule 1.6.3 from 1.6.2 Serverspec tests fail with:

```
Executing rubocop on *.rb files found in spec/.
needless argument: --debug=None
/Users/macraf/.gem/ruby/2.0.0/gems/rubocop-0.41.2/lib/rubocop/options.rb:20:in `parse'
/Users/macraf/.gem/ruby/2.0.0/gems/rubocop-0.41.2/lib/rubocop/cli.rb:24:in `run'
/Users/macraf/.gem/ruby/2.0.0/gems/rubocop-0.41.2/bin/rubocop:14:in `block in <top (required)>'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/benchmark.rb:296:in `realtime'
/Users/macraf/.gem/ruby/2.0.0/gems/rubocop-0.41.2/bin/rubocop:13:in `<top (required)>'
/usr/local/bin/rubocop:23:in `load'
/usr/local/bin/rubocop:23:in `<main>'
ERROR:

  RAN: '/usr/local/bin/rubocop spec/**/*.rb --debug=None'

  STDOUT:


  STDERR:
```

All tests which passed before now throw this error. I use Serverspec 2.36.0.

Any idea?

**Comment by retr0h** [comment]
Corrected at #227.

**Comment by retr0h** [comment]
Hi @techraf, fix has been pushed to pypi (1.7.0).
